src/3box.js

```diff
@@ -374,64 +377,6 @@ class Box extends BoxApi {
     }, [])
   }
 
-  async _writeAddressLink (proof) {
-    const validProof = await validateLink(proof)
-    if (!validProof) {
-      throw new Error('tried to write invalid link proof', proof)
-    }
-    if (await this.isAddressLinked({ address: validProof.address })) return true // address already linked
-    await this._writeRootstoreEntry(Replicator.entryTypes.ADDRESS_LINK, proof)
```

Comment:
This line writes the proof to the rootstore. Consider writing the address of the account-link to the rootstore perhaps? Later we will write the account-link docid to the "config tile".